### ISSUE TITLE
Would you like to cherrypick #15708 to Beta?

### ISSUE THREAD

**Issue by realcarlos** [flutter/flutter#15783]
Because we will release our first flutter in ID this month  , we need it very urgent , Thank you !
https://github.com/flutter/flutter/pull/15708

**Comment by chinmaygarde** [comment]
I don't think the beta channel is a separate branch. Instead, it is just a tag on the development branch. This may make it hard to apply hotfixes to the beta channel. @eseidelGoogle might have some ideas but I think you may have to apply the patch in yourself to your working copy of the framework to get yourself unblocked.

**Comment by eseidelGoogle** [comment]
Correct, we currently have no way to cherry pick to releases.  However you can always switch channels (e.g. `flutter channel dev`) or even go into your Flutter SDK directory and use direct git commands to get the exact revision you want (or cherry pick in a fix yourself).

We're working on getting the next beta out as we speak.

Finally, my understanding from @HansMuller is that our localization system is pluggable.  @HansMuller is it not possible for @realcarlos to just provide their own translations for Framework strings locally w/o them being in Flutter proper?

**Comment by HansMuller** [comment]
You could create subclasses of GlobalMaterialLocalizations and GlobalWidgetLocalizations that handled Indonesian and deferred to super for everything else. However I think it would simpler to just build with a version of Flutter that includes #15708

**Comment by realcarlos** [comment]
Ok ,thank you all ,I have forked the beta branch and made a patch for it .
